index.html

```diff
@@ -74,22 +74,79 @@
     <!-- header ends -->
 
     <!-- about starts -->
-    <section id="about" class="about">
```

Comment:
Use this section tag also along with the id so that the navbar link doesn't break.